cfg/cis-1.9/master.yaml

```diff
@@ -189,7 +189,7 @@ groups:
       - id: 1.1.13
         text: "Ensure that the default administrative credential file permissions are set to 600 (Automated)"
         audit: |
-         for adminconf in /etc/kubernetes/{admin.conf,super-admin.conf}; do if test -e $adminconf; then stat -c \"permissions=%a %n\" $adminconf; fi; done
+         for adminconf in /etc/kubernetes/{admin.conf,super-admin.conf}; do if test -e $adminconf; then stat -c "permissions=%a %n" $adminconf; fi; done
```

Comment:
alpine shell doesn't support this syntax: `{admin.conf,super-admin.conf}`

Reply:
@mozillazg I changed that to match what you proposed in the original issue.